docs/api/tuf.api.metadata.rst

```diff
@@ -4,4 +4,3 @@ Metadata
 .. automodule:: tuf.api.metadata
    :members:
    :undoc-members:
-   :show-inheritance:
```

Comment:
For almost all classes it's in my opinion not useful: either they have a `Bases: Object` line that's just noise or they identify a parent object (like Signed or BaseFile) that we don't document because it's an implementation detail that library users don't need. The only case where we use derived classes as part of the API is Serialization and there I left it visible.